kobo/apps/stripe/constants.py

```diff
@@ -4,3 +4,15 @@
     'past_due',
     'trialing',
 ]
+
+FREE_TIER_NO_THRESHOLDS = {
```

Comment:
Possibly out of scope. If there was ever a good case for type hints, this is it. Lacking type hints, a DRF serializer on the Environments endpoint would help ensure correctness. But the environments endpoint doesn't have one. IMO this should be considered in a refactor. You could ponder if the added complexity here merits adding a serializer. While not as powerful as real type hints, the serializer would provide a runtime guarantee that data is returned in the expected format. Currently it's a untyped dict that is returned, which cannot provide proof that the API consistently returns the expected format the Frontend needs to accept.